### DISCUSSION TITLE
post/eplNotAll/

### DISCUSSION THREAD

**Discussion by giscus** [WhatDamon/whatdamon.github.io#4] [General]
# post/eplNotAll/

啥是易语言易语言（EPL）是一门以中文作为程序代码编程语言，其以“易”著称，创始人为吴涛。易语言早期版本的名字为E语言，也通常代指与之对应的集成开发环境。其最早的版本的发布可追溯至2000年9月11日。创造易语言的初衷是进行用中文来编写程序的实践，方便中国人以中国人的思维编写程序，并不用再去学习西方思维。易语言的诞生极大的降低了编程的门槛和学习的难度。从2000年以来，易语言已经发展到一定的规模，

https://damon233.js.org/post/eplNotAll/

**Comment by ByYogi** [comment]
发现宝藏博客！ 页面好美观！内容质量好高！

**Comment by xbmanh** [comment]
易语言停更，好像还有一个原因就是吴涛与之前挂靠的公司的的版权纠纷问题

**Comment by yingshaoxo** [comment]
1.不跟潮流是对的，新的系统没有自由，没有最高权限。人们一般只用易语言给xp做软件。
2. 不支持x64与1G内存限制也是对的，arm都出来了，适配那些没有自由的cpu等于慢性自杀。内存上需要开多进程。
3. 至于代码保存方面，一个月存一个版本。手工硬盘存储，线上git不靠谱，随便说句f_ck就能被举报404。
4. 语言上，易语言支持纯英文代码。同时，用ascii与python通信没啥问题。都是小问题。

希望你播客能多活几年，别动不动就被举报下线或者欠费停机，best regards, see you next time！

**Reply by kamenomi-dev** [reply]
1. 潮流有一定的好处和坏处，为什么一刀切就不跟潮流？
2. 新系统怎么就没有自由？有新的规范要求和新轮子框架提供就是没自由吗？
3. x64有一定优势，甚至乎可以超越 arm。请你告诉我什么叫做没有自由的 CPU, 你要手写汇编吗？
4. 代码保存线上git怎么不靠谱？自己开发还没有规范了？为什么会被举报不想想自己做了什么？
5. 易语言的纯英文代码，如果我没记错这是靠第三方支持库——精易助手，这不就是原生不支持吗？
6. 请你回去重新看ascii是什么编码标准，中文编码是靠这个吗？你的文本操作全靠字节型吗？
